contributors/devel/sig-scalability/kubemark-outside-gce.md

```diff
@@ -0,0 +1,60 @@
+## Introduction
+This document serves to understand how to set up kubemark cluster outside gce.
```

Comment:
Why mention "outside gce"? Seems like this is a general setup instructions for many platforms (including gce).

Reply:
At the beginning,  I find some people ask how to setup kubemark cluster outside gce because existed guide is not suitable for the situation, so the original target is to help outside gce. 

In fact, this is a general setup instructions for many platforms (including gce).

I have renamed the file and updated the Introduction. Do you think it is suitable now？

Reply:
Please also change the title of this PR.

And I'd suggest removing "both inside and outside gce" and instead say say "This document serves to understand how to set up kubemark cluster given that a base cluster (to run hollow-node pods) and separate master instance(s) (to act as master(s) for the hollow nodes) are already present."

Reply:
Done